src/library_syscall.js

```diff
@@ -5,8 +5,9 @@
  */
 
 var SyscallsLibrary = {
-  $SYSCALLS__deps: ['$PATH',
+  $SYSCALLS__deps: [
```

Comment:
this removal is correct but may find existing bugs... lgtm though since the test suite at least passes, which is encouraging.